ee/codegen/src/__test__/nodes/__snapshots__/search-node.test.ts.snap

```diff
@@ -81,6 +89,133 @@ from vellum import (
     SearchResultMergingRequest,
     SearchFiltersRequest,
 )
+from vellum.types import (
+    VellumValueLogicalConditionGroupRequest,
+    VellumValueLogicalConditionRequest,
+)
+
+
+class SearchNode(BaseSearchNode):
+    query = Inputs.query
+    document_index = "d5beca61-aacb-4b22-a70c-776a1e025aa4"
+    options = SearchRequestOptionsRequest(
+        limit=8,
+        weights=SearchWeightsRequest(semantic_similarity=0.8, keywords=0.2),
+        result_merging=SearchResultMergingRequest(enabled=True),
+        filters=SearchFiltersRequest(
+            external_ids=None,
+            metadata=VellumValueLogicalConditionGroupRequest(
+                type="LOGICAL_CONDITION_GROUP",
+                combinator="AND",
+                negated=False,
+                conditions=[
+                    VellumValueLogicalConditionGroupRequest(
+                        type="LOGICAL_CONDITION_GROUP",
+                        combinator="AND",
+                        negated=False,
+                        conditions=[
+                            VellumValueLogicalConditionRequest(
+                                type="LOGICAL_CONDITION",
+                                lhs_variable=Inputs.var1,
+                                operator="=",
+                                rhs_variable=Inputs.var1,
```

Comment:
Notice that we now codegen these descriptors nested within the pydantic object. This is what's causing the new failure. Leaving like this so that we get the basic fixes in first and then we can solve this separately next.